radix-engine-interface/src/blueprints/resource/proof.rs

```diff
@@ -42,66 +38,35 @@ pub struct ProofCloneInput {}
 
 pub type ProofCloneOutput = Proof;
 
-// TODO: Evaluate if we should have a ProofValidationModeBuilder to construct more complex validation modes.
-/// Specifies the validation mode that should be used for validating a `Proof`.
-pub enum ProofValidationMode {
-    /// Specifies that the `Proof` should be validated against a single `ResourceAddress`.
-    ValidateResourceAddress(ResourceAddress),
+//========
+// Stub
+//========
 
-    /// Specifies that the `Proof` should have its resource address validated against a set of `ResourceAddress`es. If
-    /// the `Proof`'s resource address belongs to the set, then its valid.
-    ValidateResourceAddressBelongsTo(BTreeSet<ResourceAddress>),
+// TODO: update schema type
 
-    /// Specifies that the `Proof` should be validating for containing a specific `NonFungibleGlobalId`.
-    ValidateContainsNonFungible(NonFungibleGlobalId),
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct Proof(pub Own);
```

Comment:
Similarly - I'd suggest we stick on an #[sbor(transparent)] and impl all but Describe manually?

(I've still got a TODO to allow for easier replacement of validations in Describe 👍)